front/src/Administration/AnalyticsClient.ts

```diff
@@ -50,52 +50,357 @@ class AnalyticsClient {
+    }
+
+    microphone(): void {
+        this.posthogPromise
+            ?.then((posthog) => {
+                posthog.capture("wa_microphone");
+            })
+            .catch((e) => console.error(e));
+    }
+
+    settingMicrophone(value: string): void {
+        this.posthogPromise
+            ?.then((posthog) => {
+                posthog.capture("wa_setting_microphone", {
+                    checkbox: value,
+                });
+            })
+            .catch((e) => console.error(e));
+    }
+
+    settingCamera(value: string): void {
+        this.posthogPromise
+            ?.then((posthog) => {
+                posthog.capture("wa_setting_camera", {
+                    checkbox: value,
+                });
+            })
+            .catch((e) => console.error(e));
+    }
+
+    settingNotification(value: string): void {
+        this.posthogPromise
+            ?.then((posthog) => {
+                posthog.capture("wa_setting_notification", {
+                    checkbox: value,
+                });
+            })
+            .catch((e) => console.error(e));
+    }
+
+    settingFullscreen(value: string): void {
+        this.posthogPromise
+            ?.then((posthog) => {
+                posthog.capture("wa_setting_fullscreen", {
+                    checkbox: value,
+                });
+            })
+            .catch((e) => console.error(e));
+    }
+
+    settingAskWebsite(value: string): void {
+        this.posthogPromise
+            ?.then((posthog) => {
+                posthog.capture("wa_setting_ask_website", {
+                    checkbox: value,
+                });
+            })
+            .catch((e) => console.error(e));
+    }
+
+    settingRequestFollow(value: string): void {
+        this.posthogPromise
+            ?.then((posthog) => {
+                posthog.capture("wa_setting_request_follow", {
+                    checkbox: value,
+                });
+            })
+            .catch((e) => console.error(e));
+    }
+
+    settingDecreaseAudioVolume(value: string): void {
+        this.posthogPromise
+            ?.then((posthog) => {
+                posthog.capture("wa_setting_decrease_audio_volume", {
+                    checkbox: value,
+                });
+            })
+            .catch((e) => console.error(e));
+    }
+
+    login(): void {
+        this.posthogPromise
+            ?.then((posthog) => {
+                posthog.capture("wa_login");
+            })
+            .catch((e) => console.error(e));
+    }
+
+    logout(): void {
+        this.posthogPromise
+            ?.then((posthog) => {
+                posthog.capture("wa_logout");
+            })
+            .catch((e) => console.error(e));
+    }
+
+    useGoogleCalendar(): void {
+        this.posthogPromise
+            ?.then((posthog) => {
+                posthog.capture("wa_google_calendar");
+            })
+            .catch((e) => console.error(e));
+    }
```

Comment:
This is unused, right? Can we remove this?

Reply:
Oh yes, It was forgotten on my part